### PR TITLE
auth: replace domain-metadata-cache-ttl by zone-metadata-cache-ttl

### PR DESCRIPTION
**Pull request by mind04**
### Short description
and some domain -> zone in the settings descriptions and settings.rst

### Checklist
I have:
- [x] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [x] compiled this code
- [x] tested this code
- [x] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

**Comment by klaus-nicat** [comment]
Basically this makes sense. But if you go this way you also have to change the table name - TTL setting and table name IMO belong together. Or you will end up with lots renaming and have not renamed all (then better do not rename at all).

**Comment by Habbie** [comment]
> Basically this makes sense. But if you go this way you also have to change the table name - TTL setting and table name IMO belong together. Or you will end up with lots renaming and have not renamed all (then better do not rename at all).

We cannot fix everything at once. The API, for example, has always called it zone metadata. We will get there eventually.

File: pdns/pdnsutil.cc [hunk 1]
```diff
@@ -3673,7 +3675,8 @@ try
       std::map<std::string, std::vector<std::string> > meta;
       if (src->getAllDomainMetadata(di.zone, meta)) {
         for (const auto& i : meta) {
-          if (!tgt->setDomainMetadata(di.zone, i.first, i.second)) throw PDNSException("Failed to feed domain metadata");
+          if (!tgt->setDomainMetadata(di.zone, i.first, i.second))
+            throw PDNSException("Failed to feed zones metadata");
```
Comment (Habbie) [inline pdns/pdnsutil.cc:3679]:
```suggestion
            throw PDNSException("Failed to feed zone metadata");
```

**Review by Habbie — CHANGES_REQUESTED** [review]
one change requested, otherwise good!

**Review by Habbie — APPROVED** [review]

**Pull request merged**